src/storage/src/file_purger.rs

```diff
@@ -20,19 +20,23 @@ use tokio::sync::Notify;
 
 use crate::scheduler::rate_limit::{BoxedRateLimitToken, RateLimitToken};
 use crate::scheduler::{Handler, LocalScheduler, Request};
-use crate::sst::AccessLayerRef;
+use crate::sst::{AccessLayerRef, FileId};
 
 pub struct FilePurgeRequest {
     pub region_id: RegionId,
-    pub file_name: String,
+    pub file_id: FileId,
     pub sst_layer: AccessLayerRef,
 }
 
 impl Request for FilePurgeRequest {
     type Key = String;
 
     fn key(&self) -> Self::Key {
-        format!("{}/{}", self.region_id, self.file_name)
+        format!(
+            "{}/{}",
+            self.region_id,
+            self.file_id.append_extension_parquet()
```

Comment:
The `parquet` extension looks not necessary. Using the `file_id` in the key is enough.

Reply:
Ok. I'll fix it ASAP.